python/gzdice/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                gzdice
+version             0.1.2
+revision            0
+
+categories          python
+platforms           darwin
```

Comment:
`platforms darwin` is now implicit, and no longer necessary to specify in Portfiles.